crates/dojo-core/src/model/resource_metadata.cairo

```diff
@@ -4,7 +4,8 @@
 //! does not depend on dojo plugin to be built.
 //!
 use dojo::world::{IWorldDispatcherTrait, ModelIndex};
-use dojo::model::Model;
+use dojo::model::{Model, Layout, FieldLayout};
```

Comment:
We're importing `Layout` and stuff, but we actually use the fully qualified path in most of the occurrences. We may simplify the code removing the fully qualified path as we use it.

Reply:
Yes 👍